engine/base/runtime/src/main/scala/pl/touk/nussknacker/engine/baseengine/metrics/RateMeterExceptionConsumer.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.engine.baseengine.metrics
+
+import cats.data.NonEmptyList
+import pl.touk.nussknacker.engine.api.Lifecycle
+import pl.touk.nussknacker.engine.api.exception.EspExceptionConsumer
+import pl.touk.nussknacker.engine.api.runtimecontext.EngineRuntimeContext
+import pl.touk.nussknacker.engine.util.exception.GenericRateMeterExceptionConsumer
+import pl.touk.nussknacker.engine.util.metrics.{InstantRateMeterWithCount, MetricsProviderForScenario, RateMeter}
+
+class RateMeterExceptionConsumer extends GenericRateMeterExceptionConsumer with Lifecycle {
```

Comment:
Current name looks like interface that is implemented by `GenericRateMeterExceptionConsumer` - not otherwise

Reply:
I cleaned up a little bit - I've left only RateMeterExceptionConsumer for Flink with comment why it's needed there